### PR TITLE
File changes for 1.0.0.216 (#283)

### PR DESCRIPTION
**Pull request by eporres**
Some more file changes for the latest patch. Hopefully no additional editing required.

### CODE REVIEW

File: Assets/UI/Panels/notificationpanel.lua [hunk 1]
```diff
@@ -1477,6 +1488,7 @@ end
 --  All notifications are about to be refreshed
 -- ===========================================================================
 function OnNotificationRefreshRequested()
+  ClearControls();
```
Comment (Azurency) [inline Assets/UI/Panels/notificationpanel.lua]: ClearControls does not exist, it should be `ClearNotifications()`

File: Assets/UI/strategicview_mapplacement.lua [hunk 2]
```diff
@@ -218,22 +219,36 @@ function ConfirmPlaceDistrict(pInputStruct:table)
...
+            LuaEvents.CQUI_CityPanel_CityviewEnable();
+          end);
+        end
       end
-      pPopupDialog:AddCancelButton(Locale.Lookup("LOC_NO"), nil);
-      pPopupDialog:Open();
+
+      if (not IsTutorialRunning()) then
+        pPopupDialog:AddCancelButton(Locale.Lookup("LOC_NO"), nil);
+        pPopupDialog:Open();
```
Comment (Azurency) [inline Assets/UI/strategicview_mapplacement.lua]: there's an `end` missing here.

**Comment by eporres** [comment]
@Azurency I'm sorry about that. Would you like me to add a new commit or just amend the changes?

**Comment by Azurency** [comment]
You can just add a new commit ;)

File: Assets/UI/PartialScreens/citystates.lua [hunk 3]
```diff
@@ -1350,13 +1336,42 @@ function ViewCityState( iPlayer:number )
...
+
 -- ===========================================================================
 function ViewRelationships( kCityState:table )
 	-- Setup the buttons and what (sub) areas are shown/hidden.
 	Controls.EnvoysSentButton:SetSelected( false );
 	Controls.InfluencedByButton:SetSelected( false );
 	Controls.QuestsButton:SetSelected( false );
-	Controls.RelationshipsButton:SetSelected( true );
+  Controls.RelationshipsButton:SetSelected( true );
+  Controls.SingleViewStack:RegisterSizeChanged( OnSingleViewStackSizeChanged );
```
Comment (Azurency) [inline Assets/UI/PartialScreens/citystates.lua]: This `Controls.SingleViewStack:RegisterSizeChanged( OnSingleViewStackSizeChanged );` should not be here. You misplaced it, it's in `Initialize()`

**Comment by Azurency** [comment]
Thank you for your help on updating CQUI ! It's really appreciated 🎉

**Comment by eporres** [comment]
I'm glad I could help!

**Pull request merged**